### PR TITLE
Check for dist folder and index.html before running e2e tests

### PR DESCRIPTION
**Pull request by jcaianirh**
Check for dist folder and index.html before running e2e tests

### CODE REVIEW

File: test-prow-e2e.sh [hunk 1]
```diff
@@ -21,4 +21,13 @@ source ./contrib/oc-environment.sh
 kubectl create -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/okd/manifests/0.8.0/0000_30_06-rh-operators.configmap.yaml
 kubectl create -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/okd/manifests/0.8.0/0000_30_09-rh-operators.catalogsource.yaml
 
+echo "Checking for dist files:"
+while [ ! -f ./frontend/public/dist/index.html ]
+do
+ echo "#"
```
Comment (benjaminapetersen) [inline test-prow-e2e.sh]: Do we want to do something like:

```
Waiting for dist files...
Waiting...
Waiting...
Waiting...
Build complete and dist folder ready...running e2e tests
```
Just curious if `Waiting...` will be more clear than `#`.  I don't have a hugely strong opinion on this. 😄

Comment (benjaminapetersen) [inline test-prow-e2e.sh]: Second question is probably more important.  Do we want a limited number of retries?

Comment (jcaianirh) [reply test-prow-e2e.sh]: @benjaminapetersen sure that might be more clear

Comment (benjaminapetersen) [reply test-prow-e2e.sh]: I'd be curious to see how many `#` get printed this run, then perhaps do 4x as a limit, in case we want a hard fail.

Comment (benjaminapetersen) [reply test-prow-e2e.sh]: We can follow on this to clarify.  If its eliminating flakes, lets get it merged.

**Comment by benjaminapetersen** [comment]
Strange 

```

2019/02/13 18:16:12 Container teardown in pod console-e2e completed successfully
{"component":"entrypoint","level":"error","msg":"Process did not finish before 4h0m0s timeout","time":"2019-02-13T18:57:12Z"}
2019/02/13 18:57:12 error: Process interrupted with signal interrupt, exiting in 2s ...
2019/02/13 18:57:12 cleanup: Deleting release pod release-latest
2019/02/13 18:57:12 cleanup: Deleting template console-e2e
{"component":"entrypoint","level":"error","msg":"Process gracefully exited before 15s grace period","time":"2019-02-13T18:57:15Z"}
```

**Comment by spadgett** [comment]
Hey @jcaianirh. I'm not sure why this should be necessary. As far as I can tell, the commands run synchronously, so we shouldn't need to sleep. Are we sure this is the problem?

Either way, I'd think we could use `wait` instead of `sleep.`

/hold

**Comment by spadgett** [comment]
no longer needed now that we test the in-cluster console

/close

**Pull request closed without merging**